src/sass/edu-benefits.scss

```diff
@@ -25,3 +25,13 @@ select {
 legend {
   @extend h4;
 }
+
+.form-indent {
```

Comment:
The only thing is we don't have a design pattern for this on HCA now. If there's an additional question that is asked it just gets expanded below the others without being indented or anything. Do you think that would work here?